cropper/src/main/AndroidManifest.xml

```diff
@@ -12,6 +12,7 @@
         </intent>
     </queries>
     <application>
+        <activity android:name=".CropImageActivity" />
```

Comment:
This change will affect how the users consume the library?

Reply:
But is something we need? or just because of the migration? Maybe would be nice to open an issue about this for visibility?